utils/lib/dashboards.ts

```diff
@@ -0,0 +1,34 @@
+import * as glob from 'glob';
+import * as path from 'path';
+import {
+  findQuickstartComponentConfiguration,
+  FilePathAndContents,
+  readJsonFile,
+} from '../helpers';
+
+interface Dashboard {
+  name: string;
+  description: string | null;
+  pages: any;
+}
+
+export type DashboardFileAndPath = FilePathAndContents<Dashboard>;
+
+export const getAllDashboardPaths = () => {
+  return findQuickstartComponentConfiguration('dashboards');
+};
+
+// .sjon
+// .png
+// assets/iconm.svg
```

Comment:
Do we need these in here? If so, it might be good to add some more context (and fix some spelling mistakes 😄)

Reply:
Removed and included documentation for functions